spec/emites/entities/nfse_spec.rb

```diff
@@ -14,6 +131,18 @@
     :number, :rps_type, :emission_date,
     :operation_nature, :other_informations,
     :competence, :special_regime, :status,
-    :description, :send_nfse_taker, :service_values
+    :description, :send_nfse_taker, :service_values,
+    :_links
   ]
-end
+
+  describe "#url" do
+    it "returns an action endpoint" do
+      expect(subject.url("self")).to eq("https://sandbox.emites.com.br/api/v1/nfse/1")
+    end
+
+    it "returns an empty string if action is invalid" do
+      expect(subject.url("fake-action")).to eq("")
```

Comment:
Returning `nil` implies two possible return types: `NilClass` or `String`. Thats why I preferred an empty string instead of `nil`.